src/commands/LoadController.php

```diff
@@ -185,31 +185,26 @@ protected function process($data)
         }
 
         $assignData = ArrayHelper::getValue($data, 'assign', []);
-        $findBy = ArrayHelper::remove($assignData, 'findBy');
+        $findBy = ArrayHelper::remove($assignData, 'findBy', 'findIdentity');
+
+        // Check function exist before execute
+        if(!method_exists(Yii::$app->user->identityClass, $findBy)) {
+            Yii::error(sprintf("Method %s doesn't exist", $findBy));
+            exit();
+        }
+
         foreach ($assignData as $userid => $permissionOrRoles) {
```

Comment:
May be use `$userKey` instead of `$userid`, `$userID` is not always an id so we can get confuse by the name.